stagemonitor-jdbc/src/main/java/org/stagemonitor/jdbc/StagemonitorJdbcEventListener.java

```diff
@@ -135,25 +134,22 @@ public void onBeforeAnyExecute(StatementInformation statementInformation) {
 
 	@Override
 	public void onAfterAnyExecute(StatementInformation statementInformation, long timeElapsedNanos, SQLException e) {
-		final Scope activeScope = tracingPlugin.getTracer().scopeManager().active();
-		if (activeScope != null) {
-			final Span span = activeScope.span();
-			if (statementInformation.getConnectionInformation().getDataSource() instanceof DataSource && jdbcPlugin.isCollectSql()) {
-				MetaData metaData = dataSourceUrlMap.get(statementInformation.getConnectionInformation().getDataSource());
-				Tags.PEER_SERVICE.set(span, metaData.serviceName);
-				span.setTag("db.type", metaData.productName);
-				span.setTag("db.user", metaData.userName);
-
-				if (StringUtils.isNotEmpty(statementInformation.getSql())) {
-					String sql = getSql(statementInformation.getSql(), statementInformation.getSqlWithValues());
-					Profiler.addIOCall(sql, timeElapsedNanos);
-					span.setTag(AbstractExternalRequest.EXTERNAL_REQUEST_METHOD, getMethod(sql));
-					span.setTag(DB_STATEMENT, sql);
-				}
-
+		final Span span = tracingPlugin.getTracer().scopeManager().activeSpan();
```

Comment:
```suggestion
		final Span activeSpan = tracingPlugin.getTracer().scopeManager().activeSpan();
		if (activeSpan != null) {
		    ....
```
As described in JavaDoc io.opentracing.ScopeManger#activeSpan() can return null

https://github.com/opentracing/opentracing-java/blob/d91d7072e141fa66265946b5164c249bf3045017/opentracing-api/src/main/java/io/opentracing/ScopeManager.java#L64-L73
  ```
  /**
     * Return the currently active {@link Span}.
     *
     * <p>
     * Because both {@link #active()} and {@link #activeSpan()} reference the current
     * active state, they both will be either null or non-null.
     *
     * @return the {@link Span active span}, or null if none could be found.
     */
    Span activeSpan();
```
As a shortcut `tracingPlugin.getTracer().activeSpan()` should also be fine, but i don't know if that can also return null. 
This test implies that activeSpan() can return null
assertThat(tracingPlugin.getTracer().scopeManager().activeSpan()).isNull();

https://github.com/opentracing/opentracing-java/blob/d91d7072e141fa66265946b5164c249bf3045017/opentracing-api/src/main/java/io/opentracing/Tracer.java#L31-L34